src/pybind/mgr/dashboard/module.py

```diff
@@ -367,9 +367,9 @@ def set_ssl_certificate(self,
             return -errno.EINVAL, '',\
                 'Please specify the certificate file with "-i" option'
         if mgr_id is not None:
-            self.set_store('{}/crt'.format(mgr_id), inbuf)
+            self.set_store('{}/crt'.format(mgr_id), inbuf.decode())
```

Comment:
Just for keeping this 'encapsulated', there's `_get_localized_key` helper in `mgr_module` (there's also a `set_localized_store` but it only deals with the active manager):
```suggestion
            self.set_store(_get_localized_key(mgr_id, 'crt'), inbuf.decode())
```

Reply:
I created a dedicated commit to use this helper.